linera-core/src/client/mod.rs

```diff
@@ -2383,11 +2383,13 @@ where
         let committee = self.local_committee().await?;
         // Send the query to validators.
         let certificate = if round.is_fast() {
-            let hashed_value = HashedCertificateValue::new_confirmed(executed_block);
+            let hashed_value = Hashed::new(ConfirmedBlock::from_validated(ValidatedBlock::new(
+                executed_block,
+            )));
```

Comment:
```suggestion
            let hashed_value = Hashed::new(ConfirmedBlock::new(executed_block));
```

Reply:
Done in [5774032](https://github.com/linera-io/linera-protocol/pull/2988/commits/57740329ac8f5bf9d001f9899c9c59be78acda09)